docs/src/extend/custom-rules.md

```diff
@@ -192,9 +193,11 @@ The node contains all the information necessary to figure out the line and colum
 
 #### `messageId`s
 
-Instead of typing out messages in both the `context.report()` call and your tests, you can use `messageId`s instead.
+`messageId`s are the recommended alternative to typing out messages in `context.report()` calls because of the following benefits:
```

Comment:
I think this is a bit clearer.

```suggestion
`messageId`s are the recommended approach to reporting messages in `context.report()` calls because of the following benefits:
```